packages/story-editor/src/components/panels/shared/hotlinkModal.js

```diff
@@ -125,6 +163,12 @@ function HotlinkModal({
       const hotlinkInfo = await getHotlinkInfo(link);
       const shouldProxy = await checkResourceAccess(link);
 
+      if (shouldProxy && !canUseProxy) {
+        setErrorMsg(<CORSMessage />);
```

Comment:
Other error messages are in this handled in this function. https://github.com/GoogleForCreators/web-stories-wp/blob/256639d8367d95e165e38ca5e733ae0d847ff1f4/packages/story-editor/src/components/library/panes/media/local/hotlink/utils.js#L39-L55